src/modules/crud.js

```diff
@@ -0,0 +1,39 @@
+import { loadStorage, updateStorage } from './storage.js';
+import Task from './task.js';
+
+export const addTask = (index, completed, description) => {
+  const task = new Task(index, completed, description);
+  const tasks = loadStorage();
+  tasks.push(task);
+  updateStorage(tasks);
+  return loadStorage();
+};
+
+export const updateIndex = (index) => {
+  const tasks = loadStorage();
+  const updatedIndexTasks = [];
+  tasks.array.forEach((element) => {
+    if (element.id > index) {
+      element.id -= 1;
+      updatedIndexTasks.push(element.id);
+    } else {
+      updatedIndexTasks.push(element);
+    }
+  });
+  updateStorage(updatedIndexTasks);
+};
+
+export const removeTask = (index) => {
+  const tasks = loadStorage();
+  const newTasks = tasks.filter((item) => item.id !== index);
+  updateStorage(newTasks);
+  updateIndex(index);
+  return loadStorage();
+};
```

Comment:
- [ ] I can not see the option to remove a task from the to-do list, Please make sure to add an icon or an option to remove a task from the list. Please keep in mind deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

### See the image for reference 
![image](https://user-images.githubusercontent.com/83803414/159953930-7e1712a0-85a9-47fa-b328-23b95c233f5d.png)